st2common/st2common/models/db/stormbase.py

```diff
@@ -272,3 +272,17 @@ def get_reference(self):
             ref = ResourceReference(pack=self.pack, name=self.name)
 
         return ref
+
+
+class ChangeRevisionFieldMixin(object):
```

Comment:
Would be good to add some docstring to this class and clarify what it does and what kind of purpose it serves.